README.md

```diff
@@ -132,11 +132,34 @@ the [Github Pull Request Buildkite Plugin] saves the PR number with the key
 steps:
   - label: ":rubygems: Annotate Gem Changes"
     plugins:
-      - envato/bundle-update#v0.6.0:
+      - envato/bundle-update#v0.7.0:
           annotate: true
           pull-request-metadata-key: "github-pull-request-plugin-number"
 ```
 
+## Installing Custom Dependencies
+
+When running a bundle update from within a docker container, there may or may not
+be the dependencies you require for the update to complete successfully.
+For example, compiling native extensions or access to a library from another package.
+
+In this case you have 2 options to help solve the problem.
+
+1. Use a docker container which you have prebuilt (or sourced) with all the
+   required dependencies.
+
+2. You can specify a script location which will be executed prior to running the
+   bundle update. Here you can install and configure the container as needed.
+
+```yml
+steps:
+  - label: ":bundler: Update"
+    plugins:
+      - envato/bundle-update#v0.7.0:
+          update: true
+          pre-bundle-update: .buildkite/scripts/pre-bundle-update
```

Comment:
We should add this parameter to the [configuration section](https://github.com/envato/bundle-update-buildkite-plugin#configuration) of the readme too.